src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/Engine/DkmClrRuntimeInstance.cs

```diff
@@ -70,15 +70,20 @@ internal DkmClrType GetType(System.Type type)
             return new DkmClrType(module, _appDomain, (TypeImpl)type);
         }
 
-        internal DkmClrType GetType(string typeName)
+        internal DkmClrType GetType(string typeName, params System.Type[] typeArguments)
```

Comment:
`Dkm` is the common prefix for types in the Debugger API. (The ResultProvider unit tests include mocks of the necessary Debugger types.)